pkg/operator/controller/canary/daemonset_test.go

```diff
@@ -13,10 +13,11 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-func TestDesiredCanaryDaemonSet(t *testing.T) {
-	canaryImage := "openshift/hello-openshift:latest"
+// canaryImageName is the ingress-operator image
+var canaryImageName = "openshift/origin-cluster-ingress-operator:latest"
```

Comment:
```suggestion
const canaryImageName = "openshift/origin-cluster-ingress-operator:latest"
```

Reply:
Moved it to local variable in the only function it is used.